_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,341 @@
+    This can allow hidden channels---a current feature of several LN
+    implementations---to stay hidden even when routing payments from
+    arbitrary spenders.
+
+    This feature is also called rendez-vous routing.  For more
+    information, see the [description][lnrfc rz] on the LN protocol
+    documentation wiki.  See also the following mailing list threads:
+    [1][cjp rz], [2][zmn rz], [3][zmn rz packetswitch].
+
+Although discussed at the summit, the proposed 1.1 goals don't directly
+address *watchtowers* that help protect channels for users that are
+currently offline, *autopilots* that help users open their initial
+payment channels, or *deterministic preimage generation* that allow
+private keys to stay offline while an online component simply completes
+acceptance of payments.  These are services that can be built on top of
+the protocol and so don't currently require any coordination between
+implementations.
+
+## News
+
+{% comment %}<!-- math for first sub-bullet:
+exp(-45/9) # 45 minute block, 9 minute avg interval
+exp(-45/12) # 45 minute block, 12 minute avg interval
+
+The reason long interblock periods become more common at a faster rate
+than average interblock times increase is because the variance for
+independent events is the square of the average.
+
+-->{% endcomment %}
+
+- **Slowed block production, increased fees:** as widely reported,
+  several miners and mining pools are producing blocks for competing
+  forks of Bitcoin Cash when they could likely earn greater revenues by
+  creating blocks for Bitcoin.  This is the likely cause of the
+  approximately 7% reduction in Bitcoin difficulty over the retarget
+  period ending Friday (UTC) and may mean additional decreases in
+  Bitcoin hash rate and difficulty for an unknown length of time.
+  Relevant consequences for Bitcoin businesses include:
+
+    - *Slower confirmation times:* the average time between blocks may
+      increase moderately to 11 or 12 minutes and the chance of there
+      being a long wait between blocks will increase significantly in
+      relative percentages (e.g. with historically typical 9 minute
+      average block intervals, about 0.7% of blocks take more than 45
+      minutes; with a 12 minute interval, 2.3% take more than 45
+      minutes).  Recommendation: Bitcoin users are already familiar with
+      occasional long delays, so likely no action is needed.
+
+    - *Possibly increased fees:* a longer time between finding blocks
+      means less space for transactions, which can cause fee increases.
+      Occasional long waits between blocks also tend to create sudden
+      fee spikes that can persist for hours afterwards.  Recommendation:
+      ensure your fee estimation is working correctly and consider
+      preparing any fee-reduction measures you're willing to use such as
+      payment batching.
+
+    - *Increased revenues for profit-maximizing miners:* each time
```

Comment:
Note that mining becomes more profitable even before a difficulty adjustment because fees become a more significant proportion of coinbase reward (eg https://blockstream.info/tx/09ed9d39d5dab626b8d55dd0b36e63eaf15d7e19b34fa8effaae2cba8381d43c had almost 0.9BTC of fees)

Reply:
I think your comment is just saying that higher fees (described in the text's previous bullet point) mean more miner profit, ceteris paribus.  That's an obvious insight I missed, so I'm adding it.  However, the way you've phrased it makes me wonder whether I'm missing some subtler point---if so, please let me know!

Reply:
no, that's it exactly. Miners get higher fees until the downwards difficulty adjustment, and then they get more frequent blocks. Christmas has come early!